### PR TITLE
FortiGate - Fixed src adresses and dst adresses handling on get-policy

### PR DESCRIPTION
**Pull request by guykeller**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/16844

## Description
Before the fix, when calling get-policy on a policy with several destination or source addresses would only the first address would be shown. Now they are all shown

## Does it break backward compatibility?
   - No

## Must have
- [ ] Code Review

### CODE REVIEW

File: Integrations/integration-Fortigate.yml [hunk 1]
```diff
@@ -92,8 +92,6 @@ script:
         )
         if res.status_code not in {200}:
             return_error('Error in API call to FortiGate [%d] - %s' % (res.status_code, res.reason))
-        if method.upper() != 'GET':
```
Comment (ronykoz) [inline Integrations/integration-Fortigate.yml]: why?

File: Integrations/integration-Fortigate.yml [hunk 2]
```diff
@@ -410,6 +408,16 @@ script:
...
 
+    def create_addr_string(list_of_addr_data_dicts):
+        addr_string = ""
+        for addr_index in range(0, len(list_of_addr_data_dicts)):
+            cur_addr_data = list_of_addr_data_dicts[addr_index]
+            cur_addr_name = cur_addr_data.get("name")
+            if addr_index == len(list_of_addr_data_dicts) - 1:
+                addr_string += "{}".format(cur_addr_name)
+            else:
+                addr_string += "{},\n".format(cur_addr_name)
```
Comment (ronykoz) [inline Integrations/integration-Fortigate.yml]:
```suggestion
                addr_string += "{}\n".format(cur_addr_name)
```

File: Integrations/integration-Fortigate.yml (hunk 1 above)
Comment (guykeller) [reply Integrations/integration-Fortigate.yml]: It caused missing data. POST methods return data as well

**Comment by guykeller** [comment]
There are no tests for this since we have no instance, and no API documentation worth mentioning (building the integration through scraping)

File: Integrations/integration-Fortigate.yml (hunk 1 above)
Comment (ronykoz) [reply Integrations/integration-Fortigate.yml]: make sure no one uses the statuscode

**Pull request merged**